.php-cs-fixer.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()->in([__DIR__]);
+
+return (new PhpCsFixer\Config())
+    ->setRules([
+        '@Symfony' => true,
+        'ordered_imports' => true,
+        'array_syntax' => ['syntax' => 'short'],
+        'phpdoc_scalar' => false,
+    ])
+    ->setUsingCache(false)
```

Comment:
no cache? Personally I like to use it but I put the output in a `dist/` directory